go.mod

```diff
@@ -73,6 +73,7 @@ require (
 	github.com/prometheus/common v0.7.0
 	github.com/prometheus/procfs v0.0.6 // indirect
 	github.com/prometheus/prometheus v1.8.2-0.20200110114423-1e64d757f711 // master ~ v2.15.2
+	github.com/prometheus/tsdb v0.10.0
```

Comment:
This is wrong, something imports obsolete tsdb. Please check imports and move all to `github.com/prometheus/prometheus`